### DISCUSSION TITLE
Publii: Better authoring than Substack and Medium?

### DISCUSSION THREAD

**Discussion by johnwoltman** [GetPublii/Publii#1559] [General]
I was going to write an article that I didn't want to publish on my normal Publii-based website.  So I signed up for Medium and then part of the way through realized they don't support regular old vanilla HTML tables.  I tried Substack, and they don't support tables either!

So big shoutout to the Publii for getting all the basics working great.  Stuff like images and tables can be crucial for certain types of publishing.  The image gallery feature alone was what originally convinced me to move from a dynamic Django-based site to Publii.  Thank you, Publii team!

ps: and I'll end up publishing on my regular site because Publii is too smooth :D

**Comment by candidexmedia** [comment]
Agreed! There was a quick little blog that I wanted to publish, and it required embedding an Airtable iframe. I tried Medium, tumblr, etc. then gave up. It's really hard to go back to those platforms when you've experienced true flexibility.
